README.md

```diff
@@ -27,7 +29,9 @@ require(['soccer'], function (soccer) {
 Make sure you add a league table container:
 
 ```html
-<div class="league-table"></div>
+  <div class="league-container">
```

Comment:
I've just checked this again, we actually only need `<div class="league-table"></div>` and not the wrapping container div.